### ISSUE TITLE
Touchscreen not detected with official raspberry pi 7" touchscreen

### ISSUE THREAD

**Issue by ghost** [rsta2/minisynth#23]
The kernel is saying there is no touchscreen detected with the official raspberry pi 7" touchscreen on a raspberry pi 2b. 

What jumper cables have to be connected in order for this to work? I currently only have the power and ground plugged in, as I lost the other two. Do the two other optional cables have to be connected in order for minisynth to get the input?

**Comment by rsta2** [comment]
The hardware setup is described in [this article](https://www.raspberrypi.com/news/the-eagerly-awaited-raspberry-pi-display/). I tried MiniSynth Pi v5.0 again with the official 7" touchscreen on a Raspberry Pi 2B and it worked successfully. I only connected the flat 15-way cable between touchscreen and RPi and powered the touchscreen using a separate power supply, plugged into the Micro-USB jack on the touchscreen conversion board. There were no other cables.
